laravel/app/Http/Controllers/TemplateController.php

```diff
@@ -110,7 +110,7 @@ public function update(TemplateUpdateRequest $request): RedirectResponse
         try {
             $template = Template::findOrFail($request->template_id);
         } catch (ModelNotFoundException) {
-            return Redirect::route('template.edit', ['template_id' => $template->id])
+            return Redirect::route('template.edit', ['template_id' => $request->template->id])
                 ->withInput($request->all())
```

Comment:
Good catch, but I believe both will currently fail. 😅 

If the template with the ID does not exist, it will also fail to redirect to this template, so instead it should redirect to the overview as how I do it everywhere else. :)

```suggestion
            return Redirect::route('templates')
```

The alternative would be a 404 page for example, but I personally like the redirect to the overview more. :)